simulations/figure10.py

```diff
@@ -13,12 +13,13 @@
 
 vMin = 1
 
-train = Train(train='Intercity')
-
-etaMax = 0.73
-train.powerLosses = lambda f,v: f*v*(f>0)*(1 - etaMax)/etaMax - (1-etaMax)*f*v*(f<0)
-
+train = Train(config={'id':'NL_intercity_VIRM6'}, pathJSON='../trains')
 train.forceMinPn = 0
+train.forceMin = - train.forceMax
+train.powerMax = 3129277
+train.powerMin = -train.powerMax
+train.etaTraction = 0.73
+train.etaRgBrake = 0.73
```

Comment:
Why don't you use the new config feature for adding these parameters?

Reply:
It's just faster and does the same job :)